tests/integration_tests/functional/test_cpu_template_helper.py

```diff
@@ -148,6 +148,11 @@ def build_cpu_config_dict(cpu_config_path):
     # it as it is, so here we ignore subleaves 1 and 2.
     (0x12, 0x1),
     (0x12, 0x2),
+    # CPUID.18h enumerates deterministic address translation parameters and the
+    # subleaf 0 reports the maximum supported subleaf in EAX, and all the tested
+    # platforms reports 0 in EAX. However, the userspace cpuid command in ubuntu
+    # 22 also lists the subleaf 1.
```

Comment:
Is that a bug in the command, or is it because the command also lists the first invalid (sub)leaf? Or are we missing something?

Reply:
Good question... It's hard to explain, but this is basically the difference of policies about what they want to list up and I don't think it's a bug of the command.
The userspace cpuid command wants to show even the last invalid leaf in order for users to know that actually the invalid subleaves don't have meaningful values.

Reply:
It actually is a good explanation. Thanks for it.